src/views/ResetPassword.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <section id="reset-password">
+    <div class="form-container">
+      <div class="card">
+        <img
+          v-if="$config.loginLogoUrl"
+          :src="$config.loginLogoUrl"
+          class="pt-2 pb-5"
+        />
+        <div class="mb-3">
+          {{
+            $gettext(
+              "Please enter the email address associated with your account below. We'll send you an email with instructions on how to reset your password"
```

Comment:
Not needed: We'll send you an email with instructions on how to reset your password